### PR TITLE
update entities

### PR DESCRIPTION
**Pull request by BelehovEgor**
added CreatedAtUtc, CreatedBy, ModifiedAtUtc and ModifiedBy to all tables, where we need it

### CODE REVIEW

**Review by VLiamin — APPROVED** [review]

**Review by MariyaSin — APPROVED** [review]

File: src/CompanyService.Data/CompanyRepository.cs [hunk 1]
```diff
@@ -37,9 +39,11 @@ public class CompanyRepository : ICompanyRepository
         }
 
         public CompanyRepository(
-            IDataProvider provider)
+            IDataProvider provider,
+             IHttpContextAccessor httpContextAccessor)
```
Comment (Cheephot) [inline src/CompanyService.Data/CompanyRepository.cs:43]: remove space

File: src/CompanyService.Mappers/Db/DbDepartmentMapper.cs [hunk 2]
```diff
@@ -25,19 +35,22 @@ public DbDepartment Map(CreateDepartmentRequest value, Guid companyId)
                 Name = value.Name,
                 Description = value.Description,
                 IsActive = true,
+                CreatedBy = authorId,
+                CreatedAtUtc = DateTime.UtcNow,
                 Users = value.Users?.Select(du =>
-                    GetDbDepartmentUser(departmentId, du, DepartmentUserRole.Employee)).ToList()
+                    GetDbDepartmentUser(departmentId, du, authorId, DepartmentUserRole.Employee)).ToList()
+                    ?? new()
```
Comment (Cheephot) [inline src/CompanyService.Mappers/Db/DbDepartmentMapper.cs:42]: may be in 1 line?

**Review by Cheephot — APPROVED** [review]

File: src/CompanyService.Models.Db/DbDepartment.cs [hunk 3]
```diff
@@ -14,6 +14,10 @@ public class DbDepartment
         public string Name { get; set; }
         public string Description { get; set; }
         public bool IsActive { get; set; }
+        public Guid CreatedBy { get; set; }
+        public DateTime CreatedAtUtc { get; set; }
```
Comment (djino97) [inline src/CompanyService.Models.Db/DbDepartment.cs:18]: I think, not need use UTC suffix, so as utc use default everywhere

**Review by djino97 — APPROVED** [review]

**Review by Cheephot — COMMENTED** [review]
test/CompanyService.Data.UnitTests/DepartmentRepositoryTests.cs
103 117 : region - GetDepartment

File: src/CompanyService.Models.Db/DbDepartment.cs (hunk 3 above)
Comment (BelehovEgor) [reply src/CompanyService.Models.Db/DbDepartment.cs:18]: you are not right

File: src/CompanyService.Data/DepartmentRepository.cs [hunk 4]
```diff
@@ -16,10 +18,14 @@ namespace LT.DigitalOffice.CompanyService.Data
     public class DepartmentRepository : IDepartmentRepository
     {
         private readonly IDataProvider _provider;
+        private readonly IHttpContextAccessor _httpContextAccessor;
 
-        public DepartmentRepository(IDataProvider provider)
+        public DepartmentRepository(
+            IDataProvider provider,
+             IHttpContextAccessor httpContextAccessor)
```
[the remark this answers is not in the document]
Comment (BelehovEgor) [reply src/CompanyService.Data/DepartmentRepository.cs]: fixed

File: src/CompanyService.Data/CompanyRepository.cs (hunk 1 above)
Comment (BelehovEgor) [reply src/CompanyService.Data/CompanyRepository.cs]: fixed

File: src/CompanyService.Mappers/Db/DbDepartmentMapper.cs (hunk 2 above)
Comment (BelehovEgor) [reply src/CompanyService.Mappers/Db/DbDepartmentMapper.cs]: fixed

**Pull request merged**